Mlem/API/Models/Comments/APICommentReply.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  APICommentReply.swift
+//  Mlem
+//
+//  Created by Jonathan de Jong on 14.06.2023.
+//
+
+import Foundation
+
+// lemmy_db_schema::source::comment_reply::CommentReply
+struct APICommentReply: Decodable {
```

Comment:
Technically these aren't used anywhere in this PR, but i forgot to exclude them, i think it's fine though, if i rebase #145 on top of this